src/components/Dragons/Dragon.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import '../../styles/dragons.css';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveDragon, cancelReservation } from '../../redux/dragons';
+
+const Dragon = (props) => {
+  const dispatch = useDispatch();
+  const {
+    dragonName, type, dragonImg, dragonId, reserved,
+  } = props;
+  return (
+    <div className="dragon-card">
+
+      <img src={dragonImg} className="dragon-img" alt="dragon pic" />
+
+      <div className="dragon-desc">
+        <h3 className="dragon-name">
+          {dragonName}
+        </h3>
+        <p>
+          {reserved && (
+          <span className="reserve-badge">Reserved</span>
+          )}
+          {type}
+        </p>
+        {
```

Comment:
![buttons](https://user-images.githubusercontent.com/93039359/205459008-6ac494db-55d2-45d1-a213-ca25eba84562.PNG)

**[Optional]**

- [ ] Your Dragons page have some buttons for reservation which are not matching with the template which have been given to you 😨 Kindly follow the given template to make your project professional 😊